nomos-core/src/da/auth/mod.rs

```diff
@@ -0,0 +1,7 @@
+pub trait Signer {
+    fn sign(&self, message: &[u8]) -> Vec<u8>;
+}
+
+pub trait Verifier {
+    fn verify(&self, message: &[u8], sig: &[u8]) -> bool;
+}
```

Comment:
What is this used for? I am not really sure we need this.

Reply:
This will be removed as CertificateVerifier will be replaced with implementation specific verifier, but this trait was used to constraint KeyProvider:
https://github.com/logos-co/nomos-node/pull/634/files/10675e02d5317f853cc1ceb0bd9bca1d25353004#diff-bd4a1028aaa1f642e81508d453bb409463a5e4bbe6e05e8131a9e6403d756a69R7-R11